google/resource_compute_security_policy.go

```diff
@@ -53,6 +53,14 @@ func resourceComputeSecurityPolicy() *schema.Resource {
 				Description: `The project in which the resource belongs. If it is not provided, the provider project is used.`,
 			},
 
+			"type": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
```

Comment:
Should this be true?

From this it seems it is possible to update it: https://cloud.google.com/compute/docs/reference/rest/beta/securityPolicies/patch

Maybe a comment explaining why ForceNew is true is warranted.

Reply:
I've tried updating policies from edge to backend, or vice versa. Neither has worked. I've opened a case with GCP support to see if the docs are wrong, or if I've used it incorrectly

Reply:
@aucampia FYI, gcp support confirmed that it cannot be updated in place.  

> This is Shrikant from the GCP Support Networking team based in Sydney. I will be assisting you further with this case during APAC hours.
> 
> Thank you for providing a detailed description of the issue. I appreciate you taking the time to fill this out.
> 
> Looking at the equivalent gcloud command (documented at [1]), we can see that the type of the policy cannot be updated.
> Similarly, the Cloud Console UI also does not enable editing the type of the policy once created.
> 
> Therefore, I believe that this wouldn't be supported through the APIs either.
> 
> I do agree with you, that our documentation should reflect this and additionally, I believe the API response should reflect this as well.
> 
> I will reach out to the product team and keep you updated with their response.
> I hope to have an update for you within the next 48 hours.
> 
> Please let me know if you have any queries or concerns regarding the above.
> 
> Thanks and regards,
> Shrikant
> 
> [1] https://cloud.google.com/sdk/gcloud/reference/compute/security-policies/update